powerline_shell/segments/jobs.py

```diff
@@ -15,14 +29,19 @@ def run(self):
                 output_proc.communicate()[0].decode("utf-8").splitlines()[1:])
             self.num_jobs = output.count(os.getppid()) - 1
         else:
-            pppid_proc = subprocess.Popen(['ps', '-p', str(os.getppid()), '-oppid='],
-                                          stdout=subprocess.PIPE)
-            pppid = pppid_proc.communicate()[0].decode("utf-8").strip()
-            output_proc = subprocess.Popen(['ps', '-a', '-o', 'ppid'],
-                                           stdout=subprocess.PIPE)
+            pcmd = self.pidInfo(os.getppid(), "command")
+            
+            pppid = self.pidInfo(os.getppid(), "ppid")
```

Comment:
Any chance you could give `pcmd`, `pppid`, etc. more descriptive names? I know they were already like this, but I figure you might know what they mean cause I don't.

Reply:
can do